client/src/pages/Home.tsx

```diff
@@ -1,26 +1,38 @@
 import React from "react";
 import { useEffect, useState } from "react";
-import Button from "../components/Button";
 import NavigationBar from "./NavigationBar";
-
 import { addToHistory, clearHistory, postRequestOptions, retrieveHistory } from "../components/Utils";
-import DividerHorizontal from "../components/DividerHorizontal";
 
 function Home(this: any) {
     const [someState, setSomeState] = useState("not set");
-    const [textInput, setTextInput] = useState("Placeholder");
-    const [testTextInput, setTestTextInput] = useState("Insert text here to test analysis storage function");
+    const [textInput, setTextInput] = useState("not set");
```

Comment:
Should probably be initialised as an empty string

Reply:
It's purposely like this so I could see if the state was updating properly when the input was changing!